tests/run-make/link-eh-frame-terminator/rmake.rs

```diff
@@ -0,0 +1,21 @@
+// The gcc driver is supposed to add a terminator to link files, and the rustc
+// driver previously failed to do this, resulting in a segmentation fault
+// with an older version of LLVM. This test checks that the terminator is present
+// after the fix in #85395.
+// See https://github.com/rust-lang/rust/issues/47551
+
+//@ only-linux
+//@ ignore-32bit
+// Reason: the ZERO terminator will only appear when the GCC driver is used.
```

Comment:
Question: this reason doesn't seem accurate, because we are about what *rustc* does here, not what gcc driver does?

Reply:
You're right, upon re-reading it, this failure could be specific to MSVC only... let me try it on Windows GNU.